addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -23,6 +26,13 @@
  * @author Mark Hilbush - Initial contribution
  */
 public class BigAssFanHandlerFactory extends BaseThingHandlerFactory {
+    private Logger logger = LoggerFactory.getLogger(BigAssFanHandlerFactory.class);
+
+    @Override
+    protected void activate(ComponentContext componentContext) {
```

Comment:
If that's the case, in the interest of expediency, I can remove that from my PR until such time as there is a framework call to get the binding version.  :-)